blog/modules/ROOT/pages/know-your-place.adoc

```diff
@@ -0,0 +1,170 @@
+= Know Your Place
```

Comment:
Because of https://github.com/apple/pkl-lang.org/pull/46, we now have an atom feed!

And that means we need to add these attributes:

```suggestion
:author: Philip Hölzenspies
:author-link: https://github.com/holzensp
:date: <THE_CORRECT_DATE>
```

= Know Your Place
```